kit.yml

```diff
@@ -58,6 +58,12 @@ params:
     - param: bosh_network
       description: Override the default cloud-config network name for BOSH.
 
+    - param: session_timeout
+      ask: How many days would you like BOSH CLI login sessions to stay valid for?
+      description: |
+        The number of days before the BOSH CLI UAA token expires. Defaults
+        to one day.
+
```

Comment:
Let's not ask the question, and just make it a thing operators can set in their env files.